Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Build with: docker build --ulimit nofile=65535:65535 . -t custom-renovate
+# Run with: docker run --rm <additional args> custom-renovate renovate
+
+FROM quay.io/fedora/fedora:40-x86_64
+LABEL description="Mintmaker - Renovate custom image" \
+      summary="Mintmaker basic container image - a Renovate custom image" \
+      maintainer="EXD Rebuilds Guild <exd-guild-rebuilds@redhat.com >"
+
+# renovate: datasource=npm depName=renovate versioning=npm
+ARG RENOVATE_VERSION=37.407.1-custom
+
+# Using OpenSSL store allows for external modifications of the store. It is needed for the internal Red Hat cert.
+ENV NODE_OPTIONS=--use-openssl-ca
+
+RUN curl -Lo /etc/yum.repos.d/rhel7-csb-stage.repo http://hdn.corp.redhat.com/rhel7-csb-stage/rhel7-csb-stage.repo && \
+    sed -i 's/https:/http:/;s/gpgcheck=1/gpgcheck=0/' /etc/yum.repos.d/rhel7-csb-stage.repo && \
+    dnf update -y && \
+    dnf install -y \
+        redhat-internal-cert-install \
+        git \
+        python3-dnf \
+        python3-pip \
+        python3.11 \
+        python3.10 \
+        python3.9 \
+        python3.8 \
+        nodejs \
+        skopeo \
+        podman && \
+    dnf clean all && \
+    rpm --install --verbose \
+        https://github.com/tektoncd/cli/releases/download/v0.35.1/tektoncd-cli-0.35.1_Linux-64bit.rpm
+
+# Add renovate user and switch to it
+RUN useradd -lms /bin/bash -u 1001 renovate
+RUN chown -R renovate /home/renovate
```

Comment:
This is unnecessary, `useradd -m` creates the home directory with proper ownership already.